documentation/source/users/rmg/running.rst

```diff
@@ -4,33 +4,105 @@
+	  -p, --profile         run under cProfile to gather profiling statistics, and
+	                        postprocess them if job completes
+	  -P, --postprocess     postprocess profiling statistics from previous
+	                        [failed] run; does not run the simulation
+	  -t DD:HH:MM:SS, --walltime DD:HH:MM:SS
+	                        set the maximum execution time
+	  -i MAXITER, --maxiter MAXITER
+	                        set the maximum number of RMG iterations
+	  -n MAXPROC, --maxproc MAXPROC
+	                        max number of processes used during reaction
+	                        generation
+	  -k, --kineticsdatastore
+	                        output a folder, kinetics_database, that contains a
+	                        .txt file for each reaction family listing the
+	                        source(s) for each entry
+
+Some representative example usages are shown below.
+
+Run by restarting from a seed mechanism::
 
     python rmg.py input.py -r path/to/seed/
 
-Run with CPU profiling::
+Run with CPU time profiling::
 
     python rmg.py input.py -p
 
 Run with multiprocessing for reaction generation and QMTP::
 
     python rmg.py -n <Max number of processes allowed> input.py 
 
-We recommend you make a job-specific directory for each RMG simulation. Some jobs can take quite a while to complete, so we also recommend using a job scheduler if working in a linux environment. 
+Run with setting a limit on the maximum execution time::
+
+	python rmg.py input.py -t <DD:HH:MM:SS>
+
+Run with setting a limit on the maximum number of iterations::
+
+	python rmg.py input.py -i <Max number of desired iterations>
 
 
 Details on the multiprocessing implementation
-----------------------------------------------
+---------------------------------------------
 
 Currently, multiprocessing is implemented for reaction generation and the generation of QMfiles when using the QMTP option to compute thermodynamic properties of species. The processes are spawned and closed within each function. The number of processes is determined based on the ratio of currently available RAM and currently used RAM. The user can input the maximum number of allowed processes from the command line. For each reaction generation or QMTP call the number of processes will be the minimum value of either the number of allowed processes due to user input or the value obtained by the RAM ratio. The RAM limitation is employed, because multiprocessing is forking the base process and the memory limit (SWAP + RAM) might be exceeded when using too many processors for a base process large in memory.
 
 In python 3.4 new forking contexts 'spawn' and 'forkserver' are available. These methods will create new processes which share nothing or limited state with the parent and all memory passing is explicit. Once RMG is transferred to python 3 it is recommended to use the spawn or forkserver forking context to potentially allow for an increased number of processes.
```

Comment:
Not relevant to this PR, but I just noticed this comment about something that should be done "once RMG is transferred to Python 3". Is this on anyone's radar? Should it be in the issue tracker or on a kanban?

Reply:
Interesting point to bring up. It was my understanding that a previous member thoroughly explored multiprocessing options since we transitioned to python 3 awhile ago; the group has been using it for around a year I think or at least since I’ve arrived. I’m actually unsure if that paragraph is still relevant.. a quick search seems to show that both spawn and forkserver are inherently much slower than fork ([link1](https://docs.python.org/3/library/multiprocessing.html) , [link2](https://riptutorial.com/python/example/14842/the-multiprocessing-module
)), but I’ll ask around.

Reply:
Then maybe the paragraph just needs fixing/removing. Thanks for looking into it

Reply:
After digging through our [Wiki](https://github.com/ReactionMechanismGenerator/RMG-Py/wiki/Multiprocessing), I found an updated version of this paragraph:

In python 3.4 new forking contexts 'spawn' and 'forkserver' are available. These methods will create new processes which share nothing or limited state with the parent and all memory passing is explicit. Using 'spawn' or 'forkserver' in set_start_method('forkserver') raises a DatabaseError: rmgpy.exceptions.DatabaseError: Could not get database with name: kinetics. We suspect that necessary database resources are identified as unnecessary resources and therefore not inherited even though they are needed by the child process.

This result is now included in the recently pushed version of this branch. Let me know if I can help update anything else before merging 🙂

Reply:
Is the documentation really the right place to leave comments like these? I feel like this belongs somewhere else.